### PR TITLE
chore: cleanup config

### PR DESCRIPTION
**Pull request by marcoscaceres**
Depends on https://github.com/tobie/specref/pull/667

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/hr-time/pull/117.html" title="Last updated on Jun 7, 2021, 5:45 AM UTC (42c7d61)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/hr-time/117/8d554fe...42c7d61.html" title="Last updated on Jun 7, 2021, 5:45 AM UTC (42c7d61)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
References now available 🎉

**Review by yoavweiss — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -9,77 +9,45 @@
...
-        authors: [
-          'Yossef Oren',
-          'Vasileios P. Kemerlis',
-          'Simha Sethumadhavan',
-          'Angelos D. Keromytis'
-        ],
-        date: 'March 2015'
+    editors: [
+      {
+        company: "Google",
```
Comment (yoavweiss) [inline index.html:14]: Can you make this "Google LLC" while you're at it?

Comment (marcoscaceres) [reply index.html]: done

**Pull request merged**